docs/concepts/serialization.md

```diff
@@ -224,6 +224,9 @@ print(Model(x='test value').model_dump_json())
 #> {"x":"serialized test value"}
 ```
 
+!!! note A single serializer can also be called on all fields by passing the special value '*' to the [`@field_serializer`][pydantic.functional_serializers.field_serializer] decorator.
```

Comment:
I think you might want:
```suggestion
!!! note 
    A single serializer can also be called on all fields by passing the special value '*' to the [`@field_serializer`][pydantic.functional_serializers.field_serializer] decorator.
```

But I'm not entirely sure. You'll have to check the local docs preview to make sure it renders correctly!

Reply:
that's right! i checked it renders correctly this time :)